var/spack/repos/builtin/packages/whizard/package.py

```diff
@@ -99,6 +93,8 @@ def setup_build_environment(self, env):
         env.set("CXX", self.compiler.cxx)
         env.set("FC", self.compiler.fc)
         env.set("F77", self.compiler.fc)
+        env.prepend_path("LD_LIBRARY_PATH", self.spec["libtirpc"].prefix.lib)
+        env.prepend_path("CPATH", self.spec["libtirpc"].prefix.include.tirpc)
```

Comment:
We shouldn't be relying on these variables during build. Waiting for @vvolkl comments, but in case, do you have a reproducer?